src/components/app/ErrorBoundary.css

```diff
@@ -13,39 +13,19 @@
 .appErrorBoundaryContents {
   display: flex;
   width: 100%;
-  max-width: 600px;
-  max-height: calc(100%);
+  max-width: 800px;
+  max-height: 100%;
   box-sizing: border-box;
   flex-direction: column;
 }
 
-.appErrorBoundaryMessage {
-  flex: none;
-}
-
-.appErrorBoundaryDetails {
-  padding: 25px;
-  border-radius: 4px;
-  margin: 12px 0;
-  background: #fff;
-  box-shadow: 0 0 10px rgb(0 0 0 / 0.1);
-  color: var(--grey-70);
-  font-family: monospace;
-  line-height: 1.3;
-  opacity: 1;
-  overflow-y: auto;
-
-  /* Transition opacity, but do not delay changing the visibility. */
-  transition: opacity 150ms, visibility 0s;
-  visibility: visible;
-  white-space: pre-wrap;
+.appErrorBoundaryInnerText {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
 }
 
-.appErrorBoundaryDetails.hide {
-  opacity: 0;
-
-  /* Transition opacity, and create a delay on hiding the visibility to allow
-     time for the fade out. */
-  transition: opacity 150ms, visibility 0s 150ms;
-  visibility: hidden;
+.appErrorBoundaryInnerText > p {
```

Comment:
I wanted to use `p` which is semantically better, but it comes with default margin... One day we'll probably want to include some version (or our version) of normalize.css...

Reply:
Yeah, it would be good to have a normalize.css.